.vscode/c_cpp_properties.json

```diff
@@ -3,19 +3,19 @@
         {
             "name": "Linux",
             "includePath": [
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/arch/x86/include",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/arch/x86/include/generated",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/include",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/arch/x86/include/uapi",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/arch/x86/include/generated/uapi",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/include/uapi",
-                "/home/rhydon/workspace/buildroot-2020.02.4/output/build/linux-4.19.91/include/generated/uapi",
-                "/usr/lib/gcc/x86_64-linux-gnu/7/include"
+                "LINUX_SRC_PATH_PLACEHOLDER/arch/x86/include",
```

Comment:
Tell me what you think, I decided to use absolute paths that exist only on my computer because it makes it easier to change and understand which path exactly I should replace and where it is on my computer.

Reply:
I hate replacing placeholder strings like I did in `configure.py`, I'd much rather have real working paths but I don't have an alternative solution that simplifies the process.

If you want to replace the placeholder with absolute paths that'd be great. However, this is adding another commitment (make sure it works for you as the regular user would expect it to). In any case I've added what I believe is a full description of `configure.py` flag in README.md:55

Reply:
I agree with you, but this tutorial is for newbies that want to get started. I think that I prefer to separate the config file to another pr and keep the branch for other users that have more experience.

Reply:
I think the best solution is to use your absolute path and use it as a replacement constant in `configure.py`. 
Currently the manual setup exists in `README.md` it is just minimized (check it out in its rendered form). If you believe manual is better keep it that way I just think this is a tedious process.

Reply:
I like it